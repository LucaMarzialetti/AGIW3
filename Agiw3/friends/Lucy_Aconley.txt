--url_fb https://www.facebook.com/lucy.aconley
--friends
VM30061:3 Jennie Banks
VM30061:3 Amanda French
VM30061:3 Maria Zauter-Tutt
VM30061:3 Lizzie Bysouth
VM30061:3 Claire Solk
VM30061:3 Karen Newell
VM30061:3 Joel Ferguson
VM30061:3 Hannah Langley
VM30061:3 Sandi Wilkie
VM30061:3 Amanda Wallis
VM30061:3 Elspeth Heywood
VM30061:3 Chun Yip Chow
VM30061:3 Debbie Young
VM30061:3 James Simpson
VM30061:3 Samantha Jeffrey
VM30061:3 Elizabeth Collard
VM30061:3 Catherine Rudd
VM30061:3 Alana Jesurasa
VM30061:3 Sarah Manning
VM30061:3 Liz Carter
VM30061:3 Alan Briggs
VM30061:3 Leigh Cunningham
VM30061:3 Amy Gammon
VM30061:3 Victoria Nally
VM30061:3 Rishi Hindocha
VM30061:3 Hannah Draeger
VM30061:3 Dave Simpson
VM30061:3 Sarah Jayne Crosby
VM30061:3 Charlotte Thurgood
VM30061:3 Jane Batts
VM30061:3 Nicola Ferguson
VM30061:3 Macaire Bromley
VM30061:3 Rowena White
VM30061:3 Amy Rose Jex
VM30061:3 Nick Amis
VM30061:3 Virginia Tan
VM30061:3 Jason Simpson
VM30061:3 Charlotte Philo
VM30061:3 Katie Doyle
VM30061:3 Lynsey Bangert
VM30061:3 Kate Lake
VM30061:3 Meng Sun
VM30061:3 Brin Rajathurai
VM30061:3 Maryn Oppenheim
VM30061:3 Kirsten Robinson
VM30061:3 Emma Dowden-Teale
VM30061:3 Daisy Wright
VM30061:3 Alison Jensen
VM30061:3 Matt Hill
VM30061:3 Annie James
VM30061:3 Carrie-Anne Ottley
VM30061:3 Liz Radford
VM30061:3 Hannah Greaves
VM30061:3 Emma Cummings
VM30061:3 Hayley Simpson
VM30061:3 Melanie Leigh
VM30061:3 Joe Joe
VM30061:3 Suzie Wooldridge
VM30061:3 Amy Cooper-Boast
VM30061:3 Angela Minassian
VM30061:3 Gwennan Parry Thomas
VM30061:3 Candice Colbourne
VM30061:3 Catherine Sayce
VM30061:3 Jon Bond
VM30061:3 Dominique Sayers
VM30061:3 Tessa Kalaher
VM30061:3 Am√©lie Baudot
VM30061:3 Sarah McKelvie
VM30061:3 Sarah Logan
VM30061:3 Emma Sheffield
VM30061:3 Victoria Briggs
VM30061:3 Rachel Aikens
VM30061:3 Cindy Sharma
VM30061:3 Neeta Chenani
VM30061:3 Alan Curran
VM30061:3 Susie McCormack
VM30061:3 Abby Simpson
VM30061:3 Shamita Sharma
VM30061:3 Izzy Kennedy
VM30061:3 Jackie Sullivan
VM30061:3 Clare Armstrong
VM30061:3 Esther Childs
VM30061:3 Lora Dyball
VM30061:3 Jade Spinks
VM30061:3 Emma Nicholson-Swaby
VM30061:3 Douglas Graves
VM30061:3 Lynsey Gray
VM30061:3 Kim Le Rossignol
VM30061:3 Laura Perry
VM30061:3 Helen Laura Jex
VM30061:3 Lisa Matamala-Shaw
VM30061:3 Samantha Livingston
VM30061:3 Kirsty Samantha
VM30061:3 Karen Hill
VM30061:3 Nicola Duncan
VM30061:3 Claire O'Connell
VM30061:3 Sarah Smith
VM30061:3 Saira Burwood
VM30061:3 Anna Robinson
VM30061:3 Nathan Pearn
VM30061:3 Natalie McConnon
VM30061:3 Ritesh Sharma
VM30061:3 Gemma Gooch
VM30061:3 Jane Baker
VM30061:3 Melissa Manzo
VM30061:3 Robert Childs
VM30061:3 Kristina Haigh
VM30061:3 Zina Edwards
VM30061:3 Kelly Nobay
VM30061:3 Fiona Briggs
VM30061:3 Natasha Baig
VM30061:3 Sarojini Sharma
VM30061:3 Sophie Burr
VM30061:3 Ruth Alice Pearn
VM30061:3 Amie Davidson
VM30061:3 Laura Hazelton
VM30061:3 Stewart Adam Matthews
VM30061:3 Fay Amy Warwick
VM30061:3 Charlotte Haworth
VM30061:3 Simon Peace
VM30061:3 Hannah Valintine
VM30061:3 Lottie Simpson
VM30061:3 Holly Brankin
VM30061:3 Gemma Kivell
VM30061:3 Tony Rostron
VM30061:3 Robbie Simpson
VM30061:3 Matthew Simpson
VM30061:3 Hannah Davidson
VM30061:3 Nicola Simpson