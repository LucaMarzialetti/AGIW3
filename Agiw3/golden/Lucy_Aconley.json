{
    "PATTERN": {
        "tel": [
            "01582 469003",
            "01582 469003",
            "1 Valley Rise, Wheathampstead, Herts"
        ],
        "addr": [
            "AL4 8JF",
            "AL4 8JF"
        ],
        "email": ["info@ianscottphotography.co.uk"]
    },
    "NER": {
        "LOC": [
            "Wheathampstead St Albans Hertfordshire",
            "Hertfordshire",
            "England",
            "London",
            "Wales",
            "Bedfordshire",
            "Buckinghamshire"
        ],
        "ORG": [
            "Ian Scott Photography Lucy",
            "Priory Barn"
        ],
        "PER": [
            "Ian Scott",
            "Chris Aconley",
            "Lucy"
        ]
    },
    "url": "http://www.ianscottphotography.co.uk/lucy-chris-aconley-priory-barn-little-wymondley-18th-june-2010/"
}